keyboards/yushakobo/primer79/primer79.h

```diff
@@ -0,0 +1,40 @@
+/* Copyright 2022 yushakobo
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
```

Comment:
This should be moved into the info.json.   You can run `qmk info -kb yushakobo/primer79 -f json` to help with that.

Reply:
Since I am using a custom matrix.c, I could not transfer everything to json. Even if I use the json output by `qmk info -kb yushakobo/primer79 -f json`, an error occurs and I cannot build.

Reply:
The layout should still be moved, and will work, provided that the rest of the config is correct.